packages/galata/src/galata.ts

```diff
@@ -99,6 +99,13 @@ namespace galata {
     export
     type SidebarTabId = 'filebrowser' | 'jp-running-sessions' | 'tab-manager' | 'jp-property-inspector' | 'table-of-contents' | 'extensionmanager.main-view' | 'jp-debugger-sidebar';
 
+    export
+    type ToolbarItemId = 'save' | 'insert-cell-below' | 'cut-cells' | 'copy-cells' | 'paste-cells' | 'run-cell' | 'interrupt-kernel' | 'restart-kernel' | 'restart-kernel-and-run-all' | 'select-cell-type';
+
+    const ToolbarItemIdToIndexMap: Map<ToolbarItemId, number> = new Map([
+        ['save', 0], ['insert-cell-below', 1], ['cut-cells', 2], ['copy-cells', 3], ['paste-cells', 4], ['run-cell', 5], ['interrupt-kernel', 6], ['restart-kernel', 7], ['restart-kernel-and-run-all', 8], ['select-cell-type', 9]
```

Comment:
Wondering how this would play when there is for example a federated extension that adds an item in the middle of the toolbar?

Reply:
Good catch @jtpio, yes it would fail in that case. I will push an update and provide toolbar item access both by index and name/id.